gradle/lib/dependabot/gradle/file_updater/wrapper_updater.rb

```diff
@@ -49,29 +51,57 @@ def update_files(build_file)
           # If we don't have any files in the build files don't generate one
           return [] unless local_files.any?
 
+          # we only run this updater if the build file has a requirement for this dependency
+          target_requirements = dependency.requirements.select do |req|
+            T.let(req[:file], String) == build_file.name
+          end
```

Comment:
This is the where the argument for a different ecosystem starts to show up. We need to have conditional logic like this when we combine ecosystems

Not a feedback for now, it just more arguments